Structurize project according to the style guide

belgeu opened the pull request:
Since structuring of a project makes it a lot easier to navigate through project files, current project files were re-arranged to keep them in line with the following style guide - http://docs.python-guide.org/en/latest/writing/structure/ .

belgeu committed:
Structurize project according to the style guide

Since structuring of a project makes it a lot easier to navigate through project files, current project files were re-arranged to keep them in line with the following style guide - http://docs.python-guide.org/en/latest/writing/structure/ .

extsoft commented on python/pip_packages.txt:
```diff
@@ -0,0 +1,2 @@
+###### PIP packages ######
```
File name has to be "requirements.txt".

belgeu replied:
Re-named to "requirements.txt".

extsoft commented on python/tests/webdriver_test.py:
```diff
@@ -0,0 +1,21 @@
+import pytest
```
Must be in `python/test_sample/`. In this package we have only unit tests.
The same about `python/tests/unit_test.py` file.

belgeu replied:
Re-placed to `python/test_sample/` directory.

belgeu committed:
Move tests' modules to repository root (test_sample)

Package contains unit tests only. Main project modules (tests) should be placed to repository's root folder.
